### DISCUSSION TITLE
Landing Page

### DISCUSSION THREAD

**Discussion by janash** [Descriptor-Libraries/descriptor-libraries-framework#25] [Ideas]
I think PubChem has a good landing page - maybe we could consider adopting a similar layout for landing. It starts with a menu bar, then a search area, then "What is PubChem?"

![pubchem_landing](https://user-images.githubusercontent.com/18010556/230636188-2a1680d1-15fa-47b3-bbdc-d8e4d3923b35.png)

**Comment by ujamshed** [comment]
I agree! I think having the search on this landing page direct you to all the information for a specific molecule in the DB makes the most sense. When we extend to other DB's we could also have a parameter where you decide which DB you want to look up.
